core/src/main/kotlin/org/evomaster/core/output/naming/RestActionTestCaseNamingStrategy.kt

```diff
@@ -105,6 +109,39 @@ open class RestActionTestCaseNamingStrategy(
         return if (parentPathQualifier == "/") "" else getPath(parentPathQualifier)
     }
 
+    /*
+     * If any test uses query prams, then add withQueryParam(s) after the path to them to make them differ.
+     * The filter call ensures that we are only performing this disambiguation when there's only one individual that
+     * differs and the list of query params is not empty.
+     */
+    private fun checkForQueryParams(duplicatedIndividuals: MutableSet<EvaluatedIndividual<*>>): Map<EvaluatedIndividual<*>, String> {
```

Comment:
i guess could be specified `<RestIndividual` instead of `<*>`?

Reply:
Tried doing so, the Kotlin type system didn't like it much